example/storybook/helpers/CenterView.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { View, ViewStyle } from 'react-native';
+import { Theme, useStyles } from 'src';
+
+interface Props {
+  children: React.ReactNode;
+  styles?: CenterViewStyles;
+}
+
+export function CenterView({ children, styles }: Props) {
+  const { styles: $ } = useStyles('CenterView', defaultStyles, styles);
+
+  return <View style={$.container}>{children}</View>;
+}
+
+const defaultStyles = (_theme: Theme) => {
+  const container: ViewStyle = {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+  };
+
+  return {
+    container,
+  };
+};
+
+declare module '@styles' {
+  interface ComponentStyles
+    extends ComponentNamedStyles<'CenterView', typeof defaultStyles> {}
+}
+
+export type CenterViewStyles = NamedStylesProp<typeof defaultStyles>;
```

Comment:
You're right, it doesn't.  The use of the `useStyles` hook currently requires it, however, we're not actually passing custom styles in at the moment, so this might be YAGNI.

Fixed.